src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -208,6 +208,24 @@ public static void RoundtripString()
         Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
     }
 
+    [Fact]
+    public static void ToString_ByteArrayToLong_Throws()
+    {
+        byte[] arr;
+        try
+        {
+            arr = new byte[int.MaxValue / 3 + 1];
+        }
+        catch (OutOfMemoryException)
+        {
+            // Exit out of the test if we don't have an enough contiguous memory
```

Comment:
Ah, you're right. I remember reading about it but it seems it's either an extension or was removed in 2.0.